docs/source/backup/index.rst

```diff
@@ -44,9 +44,14 @@ Process
 The backup procedure consists of multiple steps executed sequentially.
 
 #. **Snapshot** - Take a snapshot of data on each node (according to backup configuration settings).
-#. (Optional) **Schema** - Upload the schema in CQL text format to the backup storage destination,
+
+   Note that ScyllaDB Manager stops `tablets <https://opensource.docs.scylladb.com/stable/architecture/tablets.html>`_  migration for the duration of this step.
```

Comment:
Maybe   `Note that ScyllaDB Manager halts `tablets <https://opensource.docs.scylladb.com/stable/architecture/tablets.html>`_ migration during this step.` ?

halts instead of stops